<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW2</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW3</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW4</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW6</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_8">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_9">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Final</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_10">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_10">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Grade</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheme A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheme B</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PIC10B. Intermediate Programming. </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PIC10C. Advanced Programming. </string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW8</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_11">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grading Scheme:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Project</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>676</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>emit_final_grade(QString)</signal>
   <receiver>textBrowser</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>785</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_change_class(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>810</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>816</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>emit_final_grade(QString)</signal>
  <slot>slot_compute()</slot>
  <slot>slot_change_class(int)</slot>
 </slots>
</ui>
